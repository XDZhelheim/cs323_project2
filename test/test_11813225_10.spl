int arrays()
{
  int a[10], b;
  float c;
  c = b[5];
  f(a[1]);
}