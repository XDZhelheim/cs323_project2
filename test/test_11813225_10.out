Error type 10 at Line 5: applying indexing operator on non-array type variables
Error type 5 at Line 5: unmatching types appear at both sides of the assignment operator
Error type 2 at Line 6: a function is invoked without a definition
